tests/test_image.py

```diff
@@ -2296,93 +2311,94 @@ def test_complex_image():
-            assert im1.conjugate()(x,y) == np.conjugate(value)
-
-            # complex conjugate is not a view into the original.
-            assert im2_conj(x,y) == 23
-            assert im2.conjugate()(x,y) == np.conjugate(value)
-
-            value2 = 10*x + y + 20j*x + 2j*y
-            im1.setValue(x,y, value2)
-            im2_view.setValue(x=x, y=y, value=value2)
-            assert im1(x,y) == value2
-            assert im1.view()(x,y) == value2
-            assert im1.view(make_const=True)(x,y) == value2
-            assert im2(x,y) == value2
-            assert im2_view(x,y) == value2
-            assert im2_cview(x,y) == value2
-
-            assert im1.real(x,y) == value2.real
-            assert im1.view().real(x,y) == value2.real
-            assert im1.view(make_const=True).real(x,y) == value2.real
-            assert im2.real(x,y) == value2.real
-            assert im2_view.real(x,y) == value2.real
-            assert im2_cview.real(x,y) == value2.real
-            assert im1.imag(x,y) == value2.imag
-            assert im1.view().imag(x,y) == value2.imag
-            assert im1.view(make_const=True).imag(x,y) == value2.imag
-            assert im2.imag(x,y) == value2.imag
-            assert im2_view.imag(x,y) == value2.imag
-            assert im2_cview.imag(x,y) == value2.imag
-            assert im1.conjugate()(x,y) == np.conjugate(value2)
-            assert im2.conjugate()(x,y) == np.conjugate(value2)
-
-            rvalue3 = 12*x + y
-            ivalue3 = x + 21*y
-            value3 = rvalue3 + 1j * ivalue3
-            im1.real.setValue(x,y, rvalue3)
-            im1.imag.setValue(x,y, ivalue3)
-            im2_view.real.setValue(x,y, rvalue3)
-            im2_view.imag.setValue(x,y, ivalue3)
-            assert im1(x,y) == value3
-            assert im1.view()(x,y) == value3
-            assert im1.view(make_const=True)(x,y) == value3
-            assert im2(x,y) == value3
-            assert im2_view(x,y) == value3
-            assert im2_cview(x,y) == value3
-            assert im1.conjugate()(x,y) == np.conjugate(value3)
-            assert im2.conjugate()(x,y) == np.conjugate(value3)
-
-    # Check view of given data
-    im3_view = galsim.Image((1+2j)*ref_array.astype(complex))
-    slice_array = (large_array * (1+2j)).astype(complex)[::3,::2]
-    im4_view = galsim.Image(slice_array)
-    for y in range(1,nrow+1):
-        for x in range(1,ncol+1):
-            assert im3_view(x,y) == 10*x + y + 20j*x + 2j*y
-            assert im4_view(x,y) == 10*x + y + 20j*x + 2j*y
-
-    # Check picklability
-    do_pickle(im1)
-    do_pickle(im1_view)
-    do_pickle(im2)
-    do_pickle(im2_view)
-    do_pickle(im3_view)
-    do_pickle(im4_view)
+    for dtype in [np.complex64, np.complex128]:
+        # Some complex modifications to tests in test_Image_basic
+        im1 = galsim.Image(ncol, nrow, dtype=dtype)
+        im1_view = im1.view()
+        im1_cview = im1.view(make_const=True)
```

Comment:
It looks like there isn't any use for it in the bulk of the test.  But I added a test that it pickles correctly at the end, which seemed to be missing.